src/components/form/FormButton/FormButton.js

```diff
@@ -0,0 +1,9 @@
+import { StyledFormButton } from "./FormButton.styled";
+
+export const FormButton = ({ type, handleClick, children }) => {
```

Comment:
It doesn't have to be named `FormButton` it can just be `Button` - component which can be use accross the project, with same styles (different for a given type of action performed by him).